mkdocs.yml

```diff
@@ -8,6 +8,7 @@ theme:
   name: material
   icon:
     repo: fontawesome/brands/github
+    logo: material/car-sports
```

Comment:
This is probably something we should change to something we own sooner rather than later. It's not great to be using something off-the-shelf for a project logo, even if the license permits.

@mariushart can we ask someone in ERST to do this? Do you know anyone that could knock something together on short order?

Reply:
Right. Just so you know, without this line, the default comes in and we were using default mkdocs's logo so far
![Screenshot from 2023-01-27 13-02-24](https://user-images.githubusercontent.com/45496492/215082711-545c3b3f-0423-4122-9e79-77117a416492.png)
so I thought it's better to set it to something more fitting.

As you said material icons are under permissive [Apache 2.0 License](https://github.com/google/material-design-icons/blob/master/LICENSE)

Reply:
Agree it's better than the default. :) It's still valuable having our own. It gives us something to defend if RT becomes too successful.